src/parsers/equipment.ts

```diff
@@ -152,6 +152,9 @@ export class EquipmentParser {
                 option.flame.allStatP = flame[1];
                 continue;
             }
+            if(name.startsWith('착용 레벨') && statNode.text.includes('-')) {
+                option.flame.reqLevel = parseInt(statNode.text.trim());
+            }
```

Comment:
```suggestion
            if (name.startsWith('착용 레벨') && statNode.text.includes('-')) {
                option.flame.reqLevel = parseInt(statNode.text.trim());
                continue;
            }
```
`continue` 넣어야 다음 if문에서 덮어써지지 않을 것 같아요

```js
// console.log(option.flame);
{
  luk: 48,
  jump: 5,
  allStatP: 5,
  reqLevel: undefined // if (STAT_MAPPING[name]) 문에 들어가서덮어써짐
}
```